### PR TITLE
[13.0][IMP] hr_attendance_reason: Add required reason on attendance screen option

### PR DESCRIPTION
**Pull request by victoralmau**
Add required reason on attendance screen option (related to https://github.com/OCA/hr-attendance/pull/96)

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT40897

### CODE REVIEW

File: hr_attendance_reason/models/hr_employee_base.py [hunk 1]
```diff
@@ -10,3 +10,6 @@ class HrEmployeeBase(models.AbstractModel):
     show_reasons_on_attendance_screen = fields.Boolean(
         related="company_id.show_reasons_on_attendance_screen", store=True
     )
+    required_reason_on_attendance_screen = fields.Boolean(
+        related="company_id.required_reason_on_attendance_screen", readonly=False
```
Comment (chienandalu) [inline hr_attendance_reason/models/hr_employee_base.py]: It doesn't make sense to be `readonly=False`, isn't it?
```suggestion
        related="company_id.required_reason_on_attendance_screen"
```

**Review by chienandalu — APPROVED** [review]
:smile: :+1:

**Comment by victoralmau** [comment]
Ping @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**